### PR TITLE
Extra Curricular User Story

### LINKED ISSUE #4: Extra Curricular
- [x] #8
- [x] #9

### PR DESCRIPTION
**Pull request by ajstyle2410**
marge courses and certification and  achievements tasks 
closes #4

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* Resetting default browser styles */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Body styling */
+body {
```
Comment (ajayg2808) [inline css/index.css]: Do not add body styling, It might impact other body tags.

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ul {
+    list-style-type: disc;
+    margin-top: 20px;
+}
+
+ul li {
+    margin-bottom: 10px;
+    padding-left: 10px;
+    font-size: 1.2em;
+}
```
Comment (ajayg2808) [inline css/index.css]: Do not use tag selector, instead prefer class/ id selectors

File: index.html [hunk 3]
```diff
@@ -13,6 +13,31 @@
 </head>
 
 <body>
+    <div class="resume-container">
```
Comment (ajayg2808) [inline index.html]: Both sections are independent, Do not enclose them in a single <div>. It is problematic for other sections.

File: index.html [hunk 4]
```diff
@@ -13,6 +13,31 @@
...
+        <!-- Courses and Certificates Section -->
+        <section class="courses-certificates">
+            <h2>Courses and Certificates</h2>
```
Comment (ajayg2808) [inline index.html]: Section title should be **"Courses and Certifications"**

File: index.html [hunk 5]
```diff
@@ -13,6 +13,31 @@
...
+            </ul>
+        </section>
+        <!-- Achievements Section -->
+        <section class="achievements">
+            <h2>Achievements</h2>
+            
+            <!-- Static List of Achievements -->
+            <ul id="achievementsList">
+                <li>Completed a Java Full-Stack Development course</li>
+                <li>Developed a Library Management System using Java, JSP & Servlets, MySQL</li>
```
Comment (ajayg2808) [inline index.html]: These are not an achievements.

**Review by ajayg2808 — CHANGES_REQUESTED** [review]
Do changes as requested.

File: css/index.css (hunk 2 above)
Comment (ajstyle2410) [reply css/index.css]: update the selector tags

File: index.html (hunk 3 above)
Comment (ajstyle2410) [reply index.html]: remove the  resume-container class

**Comment by ajstyle2410** [comment]
@ajayg2808 
update the #32  please check it

File: css/index.css [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+/* Container for the resume section */
+.courses-certificates, .achievements {
+    width: 60%;
+    margin: 10px auto;
+    background: #fff;
+    padding: 20px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.courses-certificates h2, .achievements  h2 {
```
Comment (ajayg2808) [inline css/index.css]: Create a single class instead.

File: css/index.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    font-size: 2em;
+    border-bottom: 2px solid #333;
+    padding-bottom: 10px;
+    margin-bottom: 20px;
+}
+
+/* List styling */
+.courses-certificates ul, .achievements ul {
```
Comment (ajayg2808) [inline css/index.css]: Create a single class

File: css/index.css [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    list-style-type: disc;
+    margin-top: 20px;
+}
+
+.courses-certificates ul li, .achievements ul li {
```
Comment (ajayg2808) [inline css/index.css]: Create a single class

File: index.html [hunk 9]
```diff
@@ -13,6 +13,30 @@
 </head>
 
 <body>
+        <!-- Courses and Certificates Section -->
+        <section class="courses-certificates">
+            <h2>Courses and Certificates</h2>
```
Comment (ajayg2808) [inline index.html]: Correct section title as requested in previous review

**Review by ajayg2808 — CHANGES_REQUESTED** [review]
Do changes as requested

File: index.html (hunk 4 above)
Comment (ajstyle2410) [reply index.html]: rename the title name

File: css/index.css (hunk 6 above)
Comment (ajstyle2410) [reply css/index.css]: Create a single class

File: index.html (hunk 9 above)
Comment (ajstyle2410) [reply index.html]: update the title

File: css/index.css (hunk 8 above)
Comment (ajstyle2410) [reply css/index.css]: update the in single class

File: index.html [hunk 10]
```diff
@@ -13,6 +13,30 @@
...
+                <li>Full-Stack Web Development - Udemy</li>
+            </ul>
+        </section>
+        <!-- Achievements Section -->
+        <section class="extra-curricular">
+            <h2>Achievements</h2>
+            
+            <!-- Static List of Achievements -->
+            <ul id="achievementsList">
+                <li>Completed a Java Full-Stack Development course</li>
```
Comment (ajayg2808) [inline index.html]: This is not an achievement.

File: index.html [hunk 11]
```diff
@@ -13,6 +13,30 @@
...
+                <li>Developed a Library Management System using Java, JSP & Servlets, MySQL</li>
```
Comment (ajayg2808) [inline index.html]: This is not an achievement

**Review by ajayg2808 — CHANGES_REQUESTED** [review]

File: index.html (hunk 10 above)
Comment (ajstyle2410) [reply index.html]: update the achivements

File: index.html (hunk 11 above)
Comment (ajstyle2410) [reply index.html]: update the list

File: css/index.css [hunk 12]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (ajayg2808) [inline css/index.css:1]: Padding and margins are not responsive

File: index.html
Comment (ajayg2808) [inline index.html:1]: Code formatting is not correct

**Review by ajayg2808 — APPROVED** [review]

**Pull request merged**